### ISSUE TITLE
[BUG] UI sometimes unreadable unless I switch theme after launch

### ISSUE THREAD

**Issue by Enivex** [marticliment/WingetUI#1566]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have tested that this issue has not been fixed in the latest [(beta or stable) release](https://github.com/marticliment/WingetUI/releases/).
- [X] I have checked the [FAQ](https://github.com/marticliment/WingetUI#frequently-asked-questions) section for solutions
- [X] This issue is about a bug (if it is not, please use the [correct template](https://github.com/marticliment/WingetUI/issues/new/choose))

### Describe your issue

The colors in the window are off, making the UI pretty much unreadable.

This happens sometimes when I restart the app (possibly only after reboot?), regardless of what theme I have selected. By changing the theme to something else the UI suddenly looks normal again.

"Reset WingetUI" does not fix the issue

### WingetUI Log

```text
There's nothing of interest in the log.
```

### Package Managers Logs

```text
There's nothing of interest in the log.
```

### Screenshots and videos

![image](https://github.com/marticliment/WingetUI/assets/2191754/d5f0b0c4-6e1c-4edf-bf1e-e475ad0b8b72)

![image](https://github.com/marticliment/WingetUI/assets/2191754/f4eb67cc-930f-4892-8a6b-b9ff72d3eae5)

**Comment by marticliment** [comment]
Do you usually change your windows accent color or color scheme?

**Comment by Enivex** [comment]
> Do you usually change your windows accent color or color scheme?

Not manually, but when you mention it I did recently switch to the "Windows spotlight" background with automatic accent color, and showing accent color on start, taskbar, title bars and window borders. So while I don't manually change it, it'll change quite often.

**Comment by Enivex** [comment]
So I guess this is a duplicate of https://github.com/marticliment/WingetUI/issues/1407 then
